ui/packages/catalog-ui-search/src/main/webapp/react-component/location/gazetteer.js

```diff
@@ -133,3 +216,4 @@ class Gazetteer extends React.Component {
 }
 
 module.exports = Gazetteer
+module.exports.getLargestBbox = getLargestBbox
```

Comment:
✏️ 	This syntax seems a little funky. I thought we use the object syntax if a module exports more than one thing (not sure of the exact syntax but something like this):
```
module.exports = {
	Gazetteer,
	getLargestBbox
}
```